frontend/src/components/dialogs/SecretDialogOpenstack.vue

```diff
@@ -48,67 +48,79 @@ SPDX-License-Identifier: Apache-2.0
         ></v-text-field>
       </div>
       <div>
-        <hint-colorizer hint-color="primary">
-          <v-text-field
+        <v-select
           color="primary"
-          v-model="username"
-          label="Technical User"
-          :error-messages="getErrorMessages('username')"
-          @input="$v.username.$touch()"
-          @blur="$v.username.$touch()"
-          hint="Do not use personalized login credentials. Instead, use credentials of a technical user"
-          ></v-text-field>
-        </hint-colorizer>
+          v-model="authenticationMethod"
+          label="Authentication Method"
+          :items="authenticationMethods">
+        </v-select>
       </div>
-      <div>
-        <hint-colorizer hint-color="warning">
-          <v-text-field
+      <template v-if="authenticationMethod === 'user'">
```

Comment:
maybe also indent the input fields? So that they are "grouped" under the Authentication Method